SteamTrade/SteamBotError.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace SteamTrade
+{
+    public class SteamBotError
+    {
+        SteamBotErrorType type;
```

Comment:
No access modifiers? This could be implemented as properties, rather than Java-styled getters/setters. :smile: